Solopov/DZ3-4_QuizBasicFunctional.SessionAndFilter/ServletTestList/src/main/java/ui/Account/LoginServlet.java

```diff
@@ -0,0 +1,46 @@
+package ui.Account;
+
+import services.UserService;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class LoginServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+
+
+        HttpSession session=req.getSession();
+        String login,password;
+        login=req.getParameter("uname");
+        password=req.getParameter("psw");
+        System.out.println(login+" "+password);
```

Comment:
Better use logger, instead of sout

Reply:
What kind of logger do you mean? If you use Logger.getLogger (login + "" + password); I do not see any changes in the "Tomcat LocalhostLog" section "Tomcat Catalina Log"

Reply:
Yes, log4j logger, it is better approach, than sout

Reply:
If you do not mind, I integrate log4j in the following ds since there are some usage difficulties that I would like to clarify personally